layouts/shortcodes/contributors.html

```diff
@@ -0,0 +1,32 @@
+{{ $urlPre := "https://api.github.com/repos" }}
+{{ $username := "Apache" }}
+{{ $repo := "camel-website"}}
+{{ $token := ""}}
+<!-- Enter token here -->
+{{ $scope := "all"}}
+{{ if eq $token "" }}
+<p>Unable to get contributors details[set token]</p>
+{{else}}
+{{ $collaborators := getJSON $urlPre "/" $username "/" $repo "/collaborators?access_token=" $token  }}
+
+<table class="tableblock frame-all grid-all stretch">
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Organisation</th>
+        </tr>
+    </thead>
+    <tbody>
+
+        {{range $collaborators}}
+        {{$user := getJSON .url}}
+        {{if $user.name  }}
+        <tr>
+            <td> {{$user.name}} </td>
+            <td> {{$user.company}} </td>
+        </tr>
+        {{end}}
+        {{end}}
+    </tbody>
+</table>
+{{end}}
```

Comment:
We write Hugo template directives with spaces after `{{` and before `}}`, makes it much easier to read.